pkgs/development/libraries/intel-gmmlib/default.nix

```
@@ -24,7 +26,7 @@ stdenv.mkDerivation rec {
       and buffer management for the Intel(R) Graphics Compute Runtime for
       OpenCL(TM) and the Intel(R) Media Driver for VAAPI.
     '';
-    platforms = [ "x86_64-linux" ];
+    platforms = platforms.linux;
```

```suggestion
    platforms = [ "x86_64-linux" "i686-linux" ];
```
only supports x86 linux
https://github.com/intel/gmmlib#supported-platforms